h/views/accounts.py

```diff
@@ -448,7 +441,7 @@ def post_email_form(self):
 
         self.request.user.email = appstruct["email"]
 
-        self.request.session.flash("Email address changed ✓", "success")
+        self.request.session.flash("Email address changed", "success")
```

Comment:
The tick in the message is now redundant as it is part of the message container.